src/components/overview/feedback/AddFeedback.tsx

```diff
@@ -0,0 +1,42 @@
+import { thisExpression } from "@babel/types";
+import * as React from "react";
+import { Button, Form, Input, Label, List } from "semantic-ui-react";
+import { ApiService } from "../../../services/apiService";
+
+const apiService = new ApiService();
+export default class AddFeedback extends React.Component<any, any> {
+    public state = {
+        feedback: "",
+        gottenFeedback: apiService.getFeedbackForDocumentIPFSHash(this.props.match.params.hash),
+    };
+    public handleChange = (e, data) => {
+        this.state.feedback = data.value;
+    };
+    public handleSubmit = (e, data) => {
```

Comment:
Remove unused parameters.

```suggestion
    public handleSubmit = () => {
```